robot/Cumulus/tests/browser/zrecurring_donations/zcreate_fixed_erd_record.robot

```diff
@@ -56,7 +56,10 @@ Create Fixed Recurring Donation With Monthly Installment
     ...                                     Payment Method=Credit Card
     ...                                     Day of Month=${day_of_month}
     Click Rd2 Modal Button                  Save
+
     Wait Until Modal Is Closed
+    Capture Page Screenshot
+    Sleep                                   1
```

Comment:
Did you intend to leave this sleep statement in?

Reply:
yes